src/test/java/com/zendesk/maxwell/BootstrapIntegrationTest.java

```diff
@@ -0,0 +1,259 @@
+package com.zendesk.maxwell;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.assertThat;
+
+public class BootstrapIntegrationTest extends AbstractMaxwellTest {
+	@Test
+		public void testSingleRowBootstrap() throws Exception {
+		List<RowMap> list;
+		String input[] = {
+			"insert into minimal set account_id = 1, text_field='hello'",
+			"insert into maxwell.bootstrap set database_name = 'shard_1', table_name = 'minimal'"
+		};
+		String expectedJSON[] = {
+			"{\"database\":\"shard_1\",\"table\":\"minimal\",\"type\":\"insert\",\"ts\":0,\"xid\":0,\"data\":{\"id\":0,\"account_id\":1,\"text_field\":\"hello\"}}",
+			"{\"database\":\"maxwell\",\"table\":\"bootstrap\",\"type\":\"insert\",\"ts\":0,\"xid\":0,\"data\":{\"id\":0,\"database_name\":\"shard_1\",\"table_name\":\"minimal\",\"is_complete\":0}}",
+			"{\"database\":\"shard_1\",\"table\":\"minimal\",\"type\":\"bootstrap-start\",\"ts\":0,\"data\":{}}",
+			"{\"database\":\"shard_1\",\"table\":\"minimal\",\"type\":\"insert\",\"ts\":0,\"data\":{\"id\":0,\"account_id\":1,\"text_field\":\"hello\"}}",
+			"{\"database\":\"maxwell\",\"table\":\"bootstrap\",\"type\":\"update\",\"ts\":0,\"xid\":0,\"data\":{\"id\":0,\"database_name\":\"shard_1\",\"completed_at\":\"\",\"table_name\":\"minimal\",\"started_at\":\"\",\"is_complete\":1},\"old\":{\"completed_at\":null,\"is_complete\":0}}",
+			"{\"database\":\"shard_1\",\"table\":\"minimal\",\"type\":\"bootstrap-complete\",\"ts\":0,\"data\":{}}"
```

Comment:
generally I've been liking to keep JSON as raw files on-disk, just so it doesn't look so hideous with all the backslashes.  no big deal tho.